src/app/components/action-bar/action-bar.scss

```diff
@@ -7,17 +7,19 @@
     min-height: 50px;
     z-index: 1035;
     margin-bottom: 0;
-    box-shadow: 0 -2px 6px rgba(0, 0, 0, .1);
+    box-shadow: 1px 4px 8px 1px rgba(149, 157, 165, .1);
```

Comment:
This whole file needs to go away. We need to get the styles that are necessary for the ActionBar and/or WizardBar to be in those places as `useStyles` objects, and out of this scss file. Changing this file here will impact all of the AngularJS places we're using the original action bar, and I don't know where those are

Reply:
The file doesn't have to go away now, but the functionality we need, for whatever Bar we're updating, needs to be copied / extracted / etc. into that component, and this file should be left along until we can delete it completely